crates/sui-indexer/src/handlers/handler_orchestrator.rs

```diff
@@ -197,12 +203,39 @@ impl HandlerOrchestrator {
                 );
             }
         });
+        let move_event_handle = tokio::task::spawn(async move {
```

Comment:
we could use `spawn_monitored_task` for more metrics

Reply:
great to know, will make that change with others together, thanks!